### PR TITLE
feat(react-card): bump to release candidate

### PR DESCRIPTION
**Pull request by marcosmoura**
Move card out of beta

### CODE REVIEW

File: packages/react-components/react-card/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-card",
-  "version": "9.0.0-beta.47",
+  "version": "9.0.0-rc.0",
```
Comment (Hotell) [inline packages/react-components/react-card/package.json:3]: you need to update the version also in react-components. that's why yarn failed as it tries to fetch this from npm - which is non existent

Comment (marcosmoura) [reply packages/react-components/react-card/package.json:3]: Done!

File: change/@fluentui-react-components-ccce747e-48bb-4cfe-a1e6-3b170b7c6f42.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (Hotell) [inline change/@fluentui-react-components-ccce747e-48bb-4cfe-a1e6-3b170b7c6f42.json:2]: this should be `patch` as well no ? nothing changed from stable/unstable API perspective

Comment (marcosmoura) [reply change/@fluentui-react-components-ccce747e-48bb-4cfe-a1e6-3b170b7c6f42.json:2]: Fixed

**Review by Hotell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                | `83.548 kB`<br />`23.894 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          | `78.317 kB`<br />`22.433 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |   `9.045 kB`<br />`3.806 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |  `10.984 kB`<br />`4.509 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |   `9.852 kB`<br />`4.159 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `64.09 kB`<br />`17.544 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `201.998 kB`<br />`56.27 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `35.049 kB`<br />`11.533 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `6.324 kB`<br />`2.129 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1dc9d3f16b58efb2f37e930f71b6b140953cd1ea'>1dc9d3f16b58efb2f37e930f71b6b140953cd1ea</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Avatar-mount_base.html">1630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Avatar-mount.html">1623</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Button-mount_base.html">1145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Button-mount.html">1138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Field-mount_base.html">2533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Field-mount.html">2412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProvider-mount_base.html">1951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProvider-mount.html">1928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-mount_base.html">762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-mount.html">727</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">718</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">745</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">741</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/InfoButton-mount_base.html">701</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/InfoButton-mount.html">672</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/MakeStyles-mount_base.html">2342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/MakeStyles-mount.html">2322</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Persona-mount_base.html">3790</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/Persona-mount.html">3637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/SpinButton-mount_base.html">3018</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26957/perf-test-react-components/SpinButton-mount.html">2971</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Pull request merged**